app/uk/gov/hmrc/agentclientrelationships/model/ClientRelationship.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.agentclientrelationships.model
+import play.api.libs.json.{JsPath, Json, Reads}
+import uk.gov.hmrc.domain.SaAgentReference
+
+case class ClientRelationship(agents: Seq[Agent])
```

Comment:
this case class naming doesn't sound good to me - it just holds a list of Agents , no relationship info in it
could be `case class Agents(....)` ??

Reply:
It was already there but can rename